src/main/java/frc/robot/Constants.java

```diff
@@ -57,6 +57,8 @@ public class Arm {
         // PID Profile constants
         public static final double MAXIMUM_VELOCITY = 150; // degrees per second
         public static final double MAXIMUM_ACCELERATION = 750; // degrees per second squared
+        // Speaker Shoot Angle Calculation Constants
+        public static final double SPEAKER_SHOT_ANGLE_DIF_MULTIPLIER = 0.5; // <0 -> undershoot, 0 -> bottom, 0.5 -> average, 1 -> top, >1 -> overshoot
```

Comment:
instead of this aim at the center in the lookup table and tune the single angles for each distance you go out to